apps/antalmanac/src/components/Header/SettingsMenu.tsx

```diff
@@ -1,96 +1,142 @@
-    const [theme, setTheme] = useState(AppStore.getTheme());
+function ThemeMenu() {
+    const handleThemeChange = (event: React.MouseEvent<HTMLButtonElement>) => {
+        useThemeStore.getState().setTheme(event.currentTarget.value);
+    };
 
-    const { isMilitaryTime } = useTimeFormatStore();
+    return (
+        <Box sx={{ padding: '1rem 1rem 0 1rem', width: '100%' }}>
+            <Typography variant="h6" style={{ marginTop: '1.5rem', marginBottom: '1rem' }}>
+                Theme
+            </Typography>
 
-    const handleThemeChange = () => {
-        setTheme(AppStore.getTheme());
-    };
+            <ButtonGroup style={{ display: 'flex', placeContent: 'center', width: '100%' }}>
+                <Button
+                    startIcon={<LightMode fontSize="small" />}
+                    style={{ padding: '1rem 2rem', borderRadius: '12px 0px 0px 12px', width: '100%' }}
+                    value="light"
+                    onClick={handleThemeChange}
+                >
+                    Light
+                </Button>
+                <Button
+                    startIcon={<SettingsBrightness fontSize="small" />}
+                    style={{ padding: '1rem 2rem', width: '100%' }}
+                    value="system"
+                    onClick={handleThemeChange}
+                >
+                    System
+                </Button>
+                <Button
+                    startIcon={<DarkMode fontSize="small" />}
+                    style={{ padding: '1rem 2rem', borderRadius: '0px 12px 12px 0px', width: '100%' }}
+                    value="dark"
+                    onClick={handleThemeChange}
+                >
+                    Dark
+                </Button>
+            </ButtonGroup>
+        </Box>
+    );
+}
 
-    const handleTimeChange = (event: ChangeEvent<HTMLInputElement>) => {
-        useTimeFormatStore.getState().setTimeFormat(event.target.value == 'true');
+function TimeMenu() {
+    const handleTimeChange = (event: React.MouseEvent<HTMLButtonElement>) => {
+        useTimeFormatStore.getState().setTimeFormat(event.currentTarget.value == 'true');
     };
 
-    useEffect(() => {
-        AppStore.on('themeToggle', handleThemeChange);
+    return (
+        <Box sx={{ padding: '1rem 1rem 0 1rem', width: '100%' }}>
+            <Typography variant="h6" style={{ marginTop: '1.5rem', marginBottom: '1rem' }}>
+                Time
+            </Typography>
 
-        return () => {
-            AppStore.off('themeToggle', handleThemeChange);
-        };
-    }, []);
+            <ButtonGroup style={{ display: 'flex', placeContent: 'center', width: '100%' }}>
+                <Button
+                    style={{
+                        padding: '1rem 2rem',
+                        borderRadius: '12px 0px 0px 12px',
+                        width: '100%',
+                        fontSize: '12px',
+                    }}
+                    value="false"
```

Comment:
If you have true/false, why not use a [switch](https://mui.com/material-ui/react-switch/)?
Or a single checkbox, or a single radio.

Reply:
Buttons are clearly hotter and more appealing to UCI Anteaters 🤓; It's mainly for the design choice, and MUI themselves even use buttons for they left-right / right-left setting

<img width="392" alt="Screenshot 2023-11-24 at 5 36 03 AM" src="https://github.com/icssc/AntAlmanac/assets/100006999/5226b530-6a49-4f7b-9d9f-11e66a73c695">